plugins/modules/postgresql_privs.py

```diff
@@ -1056,7 +1049,7 @@ def main():
     except TypeError as e:
         if 'sslrootcert' in e.args[0]:
             module.fail_json(msg='Postgresql server must be at least version 8.4 to support sslrootcert')
-        module.fail_json(msg="unable to connect to database: %s" % to_native(e), exception=traceback.format_exc())
+        module.fail_json(msg="Unable to connect to database: %s" % to_native(e), exception=traceback.format_exc())
```

Comment:
```suggestion
        module.fail_json(msg="unable to connect to database: %s" % to_native(e), exception=traceback.format_exc())
```
let's not mix unrelated changes in one PR

If there are any other changes that can be done in separate PRs, please remove them from here and open other PRs